substra/sdk/backends/local/compute/worker.py

```diff
@@ -135,6 +135,12 @@ def _get_cmd_template_inputs_outputs(
         ]
         return command_template
 
+    def _tmp_is_three_nine(self):
+        """Temporary helper function to remove as soon as support of Python 3.9 is dropped."""
+        import platform
+
+        return platform.python_version_tuple()[1] == "9"
+
```

Comment:
Do we have this in a Linear card or something to remember that we can get rid of this?

Reply:
Added in the dopr-Python3.9 card